scene/resources/text_paragraph.cpp

```diff
@@ -675,11 +671,7 @@ Size2 TextParagraph::get_line_size(int p_line) const {
 
 	const_cast<TextParagraph *>(this)->_shape_lines();
 	ERR_FAIL_COND_V(p_line < 0 || p_line >= (int)lines_rid.size(), Size2());
-	if (TS->shaped_text_get_orientation(lines_rid[p_line]) == TextServer::ORIENTATION_HORIZONTAL) {
-		return Size2(TS->shaped_text_get_size(lines_rid[p_line]).x, TS->shaped_text_get_size(lines_rid[p_line]).y);
-	} else {
-		return Size2(TS->shaped_text_get_size(lines_rid[p_line]).x, TS->shaped_text_get_size(lines_rid[p_line]).y);
-	}
+	return TS->shaped_text_get_size(lines_rid[p_line]);
```

Comment:
Orientation check is done in the `shaped_text_get_size`, so this change is correct.